src/types.ts

```diff
@@ -25,6 +25,7 @@ export interface Column<TRow, TSummaryRow = unknown> {
   groupFormatter?: React.ComponentType<GroupFormatterProps<TRow, TSummaryRow>>;
   /** Enables cell editing. If set and no editor property specified, then a textinput will be used as the cell editor */
   editable?: boolean | ((row: TRow) => boolean);
+  colSpan?: ((args: ColSpanArgs<TRow, TSummaryRow>) => number | undefined);
```

Comment:
```suggestion
  colSpan?: (args: ColSpanArgs<TRow, TSummaryRow>) => number | undefined;
```

Reply:
I also wonder if we should just rename it to `span`.
Otherwise it can read as redundant: `column.col(umn)Span`

Reply:
Should we change it to `cellSpan` to match `cellClass`? `colSpan` is a standard term (for tables and aria label) so I think it may be better than `span` and if we ever add `rowSpan` then it `colSpan` is better